plugins/scaffolder-backend-module-gitlab/src/actions/gitlabIssueEdit.ts

```diff
@@ -216,19 +226,36 @@ export const editGitlabIssueAction = (options: {
           weight,
         };
 
-        const response = (await api.Issues.edit(
-          projectId,
-          issueIid,
-          editIssueOptions,
-        )) as IssueSchema;
+        const editedIssue = await ctx.checkpoint({
+          key: `issue.edit.${projectId}.${issueIid}.${generateStableHash(
```

Comment:
Do we need the `stableHash` in the key here? Isn't it enough because it's scoped by `taskId` anyways?

Reply:
Hm, I was thinking about if the input will change. 

But yeah, at that case we create now a new task, then indeed, we don't need a hash, i'll remove it